nat-lab/tests/telio.py

```diff
@@ -1048,6 +1050,76 @@ async def get_log(self) -> str:
         await process.execute()
         return process.get_stdout()
 
+    async def get_system_log(self) -> Optional[str]:
+        """
+        Get the system log on the target machine
+        Windows only for now
+        """
+        if self._connection.target_os == TargetOS.Windows:
+            output_notifier = OutputNotifier()
+            done_event = asyncio.Event()
+            done_event_str = "Get-EventLog - Done!"
+
+            async def on_stdout(stdout: str) -> None:
+                for line in stdout.splitlines():
+                    output_notifier.handle_output(line)
+
+            output_notifier.notify_output(
+                f"{done_event_str}",
+                done_event,
+            )
+
+            process = self._connection.create_process([
+                "powershell",
```

Comment:
Have you tried `powershell -Command Get-EventLog....`? or if that does not work `powershell -File <some script with Get-EventLog command>`? It sounds like it should simplify things.

Reply:
@matislovas tried and couldn't manage to make it work, myself I haven't tried

Reply:
Managed to make it work with `powershell -Command Get-EventLog`, so should be more clean now